### ISSUE TITLE
`-sync y` breaks Sun GridEngine

### ISSUE THREAD

**Issue by shuoyangd** [shuoyangd/tape4nmt#16]
Should be able to fix it by hanging the job on `exitcode`, instead of waiting for `qsub` to return.

**Comment by mjpost** [comment]
Really? How does this break? That's too bad.

**Comment by shuoyangd** [comment]
On CLSP grid, after submitting large batch of jobs with `-sync y`, the following error would pop up:

```
Unable to initialize environment because of error: cannot register event client. Only 979 event clients are allowed in the system
```

According to Dan, this is a bug in qmaster, so `-sync y` should be avoided.

**Comment by shuoyangd** [comment]
resolved with a4e0b71
